pkg/apis/core/v1alpha1/types_overridepolicy.go

```diff
@@ -81,12 +81,46 @@ type TargetClusters struct {
 	ClusterAffinity []ClusterSelectorTerm `json:"clusterAffinity,omitempty"`
 }
 
+// Overriders contains a list of override patch
+// The order in which the override patches take effect is:
+// - Image
+// - JsonPatch
 type Overriders struct {
+	Image []ImageOverrider `json:"image,omitempty"`
 	// JsonPatch specifies overriders in a syntax similar to RFC6902 JSON Patch.
 	// +optional
 	JsonPatch []JsonPatchOverrider `json:"jsonpatch,omitempty"`
 }
 
+type ImageOverrider struct {
+	// ContainerName is ignored when ImagePath is set
+	// If null, the image override rule applies to all containers.
+	// If specified, it should match one container or initContainer name in pod template.
+	// +optional
+	ContainerName string `json:"containerName,omitempty"`
+
+	// ImagePath indicates the path of target image field.
+	// Defaults to empty, the system will automatically detect image fields if the resource type is
+	// Pod, CronJob, Deployment, StatefulSet, DaemonSet or Job
+	// If not empty, the image override rule applies to specify image
+	// +optional
+	ImagePath string `json:"imagePath,omitempty"`
```

Comment:
```suggestion
	// ImagePath indicates the image path to target.
	// If empty, the system will automatically detect the image path if the resource type is
	// Pod, CronJob, Deployment, StatefulSet, DaemonSet or Job.
	// +optional
	ImagePath string `json:"imagePath,omitempty"`
```

Reply:
Let's also give an example, since users might wonder whether the path should be dot or slash separated.

Reply:
"... automatically detect ..." -> "... automatically resolve ..."

Reply:
The code has been updated.